pkg/controller/common_test.go

```diff
@@ -751,3 +752,9 @@ func mockClients(mockClient *mockclient.MockClient, azVolumeClient azdisk.Interf
 		}).
 		AnyTimes()
 }
+
+func addTestNodeInAvailableAttachmentsMap(sharedState SharedState, nodeName string) {
```

Comment:
The `sharedState` parameter needs to be a pointer.

Reply:
Done.